### PR TITLE
Add IsProxy on RpcFunctionMetadata

### PR DESCRIPTION
**Pull request by pragnagopa**

### CODE REVIEW

**Comment by Francisco-Gamino** [comment]
One minor comment/question. Other than that, LGTM.

**Comment by pragnagopa** [comment]
@Francisco-Gamino  - i do not see any unresolved comments. Can you please add your comment?

**Comment by Francisco-Gamino** [comment]
> @Francisco-Gamino - i do not see any unresolved comments. Can you please add your comment?

@pragnagopa : comment added.

File: src/WebJobs.Script.Grpc/azure-functions-language-worker-protobuf/src/proto/FunctionRpc.proto [hunk 1]
```diff
@@ -427,4 +430,4 @@ message RpcHttp {
   TypedData rawBody = 17;
   repeated RpcClaimsIdentity identities = 18;
   repeated RpcHttpCookie cookies = 19;
-}
+}
```
[the remark this answers is not in the document]
Comment (pragnagopa) [reply src/WebJobs.Script.Grpc/azure-functions-language-worker-protobuf/src/proto/FunctionRpc.proto]: Nope. I edited in VSCode. We can address this later in protobuf repo.

[the remark this answers is not in the document]
Comment (Francisco-Gamino) [reply src/WebJobs.Script.Grpc/azure-functions-language-worker-protobuf/src/proto/FunctionRpc.proto]: Sounds good.

**Pull request merged**